x/pocketcore/keeper/service_test.go

```diff
@@ -191,4 +201,83 @@ func TestKeeper_HandleRelay(t *testing.T) {
 		assert.Equal(t, err.Codespace(), sdk.CodespaceType(types.ModuleName))
 		assert.Equal(t, err.Code(), sdk.CodeType(types.CodeInvalidBlockHeightError))
 	}
+
+	// TC2:
+	// Test a relay while one session behind and forward,
+	// while ClientSessionSyncAllowance = 1
+	// --> Success on behind session block height
+	// --> InvalidBlockHeightError on forward session block height
+	SessionRangeTc := 1
+	types.GlobalPocketConfig.ClientSessionSyncAllowance = int64(SessionRangeTc)
+
+	// First test the minimum boundary
+	resp, err = testRelayAt(
+		t,
+		mockCtx,
+		keeper,
+		latestSessionHeight-blocksPerSesssion*int64(SessionRangeTc),
+		clientPrivateKey,
+		appPrivateKey,
+		nodePubKey,
+		chain,
+	)
+	assert.Nil(t, err, err)
```

Comment:
NIT: The error will show up if it's not nil

```suggestion
	assert.Nil(t, err)
```